tests/e2e/e2e_test.go

```diff
@@ -146,17 +146,14 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 		})
 
 		ginkgo.By("start request with invalid custom VM name format should fail", func() {
-			pluginsPath := filepath.Join(os.TempDir(), "plugins")
-			err := os.MkdirAll(pluginsPath, fs.ModePerm)
-			gomega.Ω(err).Should(gomega.BeNil())
-			f, err := os.CreateTemp(pluginsPath, strings.Repeat("a", 33))
+			f, err := os.CreateTemp(os.TempDir(), strings.Repeat("a", 33))
 			gomega.Ω(err).Should(gomega.BeNil())
 			filePath := f.Name()
 			gomega.Ω(f.Close()).Should(gomega.BeNil())
 
 			ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
 			_, err = cli.Start(ctx, execPath1,
-				client.WithBuildDir(filepath.Dir(filepath.Dir(filePath))),
+				client.WithPluginDir(filepath.Dir(filePath)),
```

Comment:
Wouldn't this actually fail due to bad plugins folder name rather than bad VM name?

Reply:
No. Start is failing only for VM binary not found.